src/main/resources/templates/tabela.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"/>
+    <title>Tabela produktów</title>
+</head>
+<body>
+<!--<div th:replace="elements :: header"></div>-->
+
+<table class="table table-sm" style="width:30%">
+    <tr>
+        <th>Produkt</th>
+        <th>Cena (zł)</th>
+    </tr>
+    <tr th:each="product: ${products}">
+        <td th:text="${product.name}">Nazwa</td>
+        <td th:text="${product.price}">Cena</td>
```

Comment:
To wszystko mogłoby się znajdować w `tbody` gdzie dodajemy ciało (główne dane) tabeli